src/E57Simple.cpp

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2010 Stan Coleby (scoleby@intelisum.com)
+ * Copyright (c) 2020 PTC Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person or organization
+ * obtaining a copy of the software and accompanying documentation covered by
+ * this license (the "Software") to use, reproduce, display, distribute,
+ * execute, and transmit the Software, and to prepare derivative works of the
+ * Software, and to permit third-parties to whom the Software is furnished to
+ * do so, all subject to the following:
+ *
+ * The copyright notices in the Software and this entire statement, including
+ * the above license grant, this restriction and the following disclaimer,
+ * must be included in all copies of the Software, in whole or in part, and
+ * all derivative works of the Software, unless such copies or derivative
+ * works are solely in the form of machine-executable object code generated by
+ * a source language processor.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE, TITLE AND NON-INFRINGEMENT. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDERS OR ANYONE DISTRIBUTING THE SOFTWARE BE LIABLE
+ * FOR ANY DAMAGES OR OTHER LIABILITY, WHETHER IN CONTRACT, TORT OR OTHERWISE,
+ * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+
+//! @file E57Simple.cpp
+
+// for M_PI. This needs to be first, otherwise we might already include math header
+// without M_PI and we would get nothing because of the header guards.
+#define _USE_MATH_DEFINES
+#include <cmath>
+
+#include "E57Simple.h"
+#include "E57SimpleImpl.h"
+
+using namespace e57;
+using namespace std;
```

Comment:
I would prefer not to do this. `using namespace std;` is discouraged.

(I understand it is in the older code, but I'd prefer not to add it to new code.)